### ISSUE TITLE
Setting for detecting problems on file save rather than continually

### ISSUE THREAD

**Issue by briankendall** [microsoft/vscode#74157]
Currently when editing C# or C++ files, VS Code will automatically detect errors in the code as I type. I find this to be quite distracting as it will report errors in lines or code blocks that are incomplete, and where I'm well aware there's an error.

As an alternative, I'd like it to be possible for VS Code to detect problems when I save the current file, since I typically save once an edit is complete and I believe the code to be in a syntactically correct state. Alternatively, being able to detect problems via an explicit command (perhaps via a keystroke) would work well too.

**Comment by usernamehw** [comment]
There is an issue from the past that mentions delay or showing decorations on save #63737. But it was closed as `extension-candidate` .

**Comment by sandy081** [comment]
Yeah, this is a duplicate of #63737. I do not think there are any plans to build this feature into VS Code but as said this can be requested to be incorporated in the extension that generates these errors/warnings
